helix-core/src/test/java/org/apache/helix/integration/TestPartitionMovementThrottle.java

```diff
@@ -279,13 +332,15 @@ public void testResourceThrottleWithDelayRebalancer() {
     Assert.assertTrue(_clusterVerifier.verifyByPolling());
 
     for (String db : _dbs) {
-      validateThrottle(DelayedTransition.getResourcePatitionTransitionTimes(), db, 2);
+      int maxInParallel = getMaxParallelTransitionCount(DelayedTransition.getResourcePatitionTransitionTimes(), db);
+      System.out.println(
+          "MaxInParallel: " + maxInParallel + " maxPendingTransition: " + 2);
+      Assert.assertTrue(maxInParallel <= 2, "Throttle condition does not meet for " + db);
     }
   }
 
-  private void validateThrottle(
```

Comment:
Is there a strong reason we're modifying this method?

Reply:
Yes. The TestHelper.verify method expects an interface with boolean return value. The original method returns void. As you can see, I just pull out the part where it verifies `maxParallelCount <= input `which makes more sense and visible

Reply:
@i3wangyi You make a great point. Thanks for clarifying and I believe this is the correct thing to do.